### ISSUE TITLE
Error executing 'postInstallation': File /opt/bitnami/sonarqube/conf/sonar.properties did not match any of the provided substitutions

### ISSUE THREAD

**Issue by vikriusman** [bitnami/bitnami-docker-sonarqube#35]
**Description**

I just started a new sonarqube using docker-compose, but I found an error like the one below, in the previous deployments I used a similar docker-compose file and the results are fine and smooth

**Steps to reproduce the issue:**

here is my docker-compose file
```

version: '3.3'

services:
  postgresql:
    image: 'bitnami/postgresql:latest'
    environment:
      - ALLOW_EMPTY_PASSWORD=yes
      - POSTGRESQL_USER=sonarqube
      - POSTGRESQL_DATABASE=sonarqube
      - POSTGRESQL_PASSWORD=pgpass
    restart: always
    volumes:
      - ./pgsonardata:/bitnami/postgresql

  sonarqube:
    image: bitnami/sonarqube:8
    depends_on:
      - postgresql
    ports:
      - '9001:9000'
    environment:
      - SONARQUBE_USERNAME=username
      - SONARQUBE_PASSWORD=password
      - SONARQUBE_DATABASE_USER=sonarqube
      - SONARQUBE_DATABASE_NAME=sonarqube
      - SONARQUBE_DATABASE_PASSWORD=pgpass
      - ALLOW_EMPTY_PASSWORD=yes
    restart: always
    volumes:
      - ./sonardata:/bitnami
```

**Describe the results you received:**

here is my error :
```
Welcome to the Bitnami sonarqube container
Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-sonarqube
Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-sonarqube/issues

nami    INFO  Initializing sonarqube
sonarqu INFO  ==> Creating server folders...
sonarqu INFO  ==> Configuring database parameters...
Error executing 'postInstallation': File /opt/bitnami/sonarqube/conf/sonar.properties did not match any of the provided substitutions
```

i dont know what really happens

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Docker version 1.13.1, build 64e9980/1.13.1
```

- Output of `docker info`:

```
Containers: 6
 Running: 3
 Paused: 0
 Stopped: 3
Images: 8
Server Version: 1.13.1
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: journald
Cgroup Driver: systemd
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: n94ome1gu5wmhb3wcfrlbtt5l
 Is Manager: true
 ClusterID: 0dgmh9bu9bfka7197n2s3kqat
 Managers: 1
 Nodes: 8
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: 192.168.50.2
 Manager Addresses:
  192.168.50.2:2377
Runtimes: runc docker-runc
Default Runtime: docker-runc
Init Binary: /usr/libexec/docker/docker-init-current
containerd version:  (expected: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1)
runc version: 66aedde759f33c190954815fb765eedc1d782dd9 (expected: 9df8b306d01f59d3a8029be411de015b7304dd8f)
init version: fec3683b971d9c3ef73f284f176672c44b448662 (expected: 949e6facb77383876aeff8a6944dde66b3089574)
Security Options:
 seccomp
  WARNING: You're not using the default seccomp profile
  Profile: /etc/docker/seccomp.json
Kernel Version: 3.10.0-1127.19.1.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 3
CPUs: 4
Total Memory: 7.777 GiB
Name: localhost.localdomain
ID: 4T5E:NUBU:VHD2:MYHG:XUXY:LQWO:OHE5:WOAH:H6FN:6OHU:XD56:NICP
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Registries: docker.io (secure)
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.24.1, build 4667896b
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
im using local virtual cloud

**Comment by marcosbc** [comment]
Hi @vikriusman, unfortunately I'm unable to reproduce your issue.

Make sure that, when mounting volumes locally, those folders have write privileges (you can try with 777 for those folders) and that they are empty before initializing.

**Comment by marcosbc** [comment]
Hi all @vikriusman, we just released [`SonarQube 9.0.0-10-debian-10-r0`](https://github.com/bitnami/bitnami-docker-sonarqube#900-debian-10-r0) which includes major changes, and the source code is now available in the [`rootfs` folder](https://github.com/bitnami/bitnami-docker-sonarqube/tree/master/9/debian-10/rootfs) in this repo.

As part of these changes, we improved the initialization logic so these types of errors should not happen anymore.
